src/xas_tdp_types.F

```diff
@@ -262,13 +275,14 @@ MODULE xas_tdp_types
       INTEGER                                 :: state_type
       INTEGER, DIMENSION(:), POINTER          :: blk_size
       REAL(dp), DIMENSION(:, :), POINTER      :: energy_evals
+      REAL(dp), DIMENSION(:, :), POINTER      :: gw2x_evals
```

Comment:
couldn't this be an allocatable instead?

Reply:
It might be. I am not sure how much the code would need to be adapted. I will look into it.